LiteFarmOrg/LiteFarm
# Bug fix/update tasks due today query to fix multiple farm bug

**Description:**
If a worker works for Farm A and Farm B. The said worker has one task due today for Farm A, but the due today endpoint will send notifications of that one task to both Farm A and B.

Solution was to use the pre-existing `getTasksForFarm()` function to retrieve only tasks associated with a given farm. Then filter out by due date and given user.

**To Test:**
1. Have a farm worker account working for two farms.
2. Create a task that's due today and assign it to the worker from step 1 on only one farm.
3. Call the endpoint (instruction is [here](https://github.com/LiteFarmOrg/LiteFarm/pull/1922))
4. You should only see the notification on the farm that has the task due today. (You might have to log out and log back in to clear the cached notifications between farms)
